spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -273,32 +273,36 @@ static void deleteExampleData(DatabaseClient dbClient) {
 
   // [START spanner_query_data]
   static void query(DatabaseClient dbClient) {
-    // singleUse() can be used to execute a single read or query against Cloud Spanner.
-    ResultSet resultSet =
+    // We use a try-with-resource block to automatically release resources held by ResultSet.
+    try (ResultSet resultSet =
         dbClient
+            // singleUse() can be used to execute a single read or query against Cloud Spanner.
```

Comment:
Strike the word and put it on the same line as the method.

Reply:
Done.